Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Init.hs

```diff
@@ -103,20 +104,22 @@ handler merchantId req = do
     throwError $ QuoteExpired driverQuote.id.getId
   searchRequest <- QSR.findById driverQuote.searchRequestId >>= fromMaybeM (SearchRequestNotFound driverQuote.searchRequestId.getId)
   -- do we need to check searchRequest.validTill?
-  booking <- buildBooking searchRequest driverQuote now
+  booking <- buildBooking searchRequest driverQuote transporter now
   Esq.runTransaction $
     QRB.create booking
   pure InitRes {..}
   where
-    buildBooking searchRequest driverQuote now = do
+    buildBooking searchRequest driverQuote merchant now = do
       id <- Id <$> generateGUID
       fromLocation <- buildBookingLocation searchRequest.fromLocation
       toLocation <- buildBookingLocation searchRequest.toLocation
+      exoPhone <- getRandomElement merchant.exoPhones
```

Comment:
As of till now we were not having multiple exophones for the users, so in exophones we were saving exophone with its backup exophone in the array. Now, you have to change that. exophones should be array of object (primary: number, backup: number)